### PR TITLE
[databuffer] Cleanup databuffers for ccapi

### PR DESCRIPTION
**Pull request by kparichay**
Cleanup databuffer headers for ccapi

See also #199

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Review by zhoonit — APPROVED** [review]
LGTM

File: nntrainer/include/databuffer_util.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file	addition_layer.h
```
Comment (jijoongmoon) [inline nntrainer/include/databuffer_util.h:5]: check filename?

File: nntrainer/src/databuffer_file.cpp [hunk 2]
```diff
@@ -51,9 +52,34 @@ extern std::condition_variable cv_train;
 extern std::condition_variable cv_val;
 extern std::condition_variable cv_test;
 
-extern DataStatus trainReadyFlag;
-extern DataStatus valReadyFlag;
-extern DataStatus testReadyFlag;
+#define SET_VALIDATION(val)                                              \
+  do {                                                                   \
```
Comment (jijoongmoon) [inline nntrainer/src/databuffer_file.cpp:56]: This already set in databuffer_util.h.

File: nntrainer/src/databuffer_func.cpp [hunk 3]
```diff
@@ -51,9 +52,34 @@ extern std::condition_variable cv_train;
 extern std::condition_variable cv_val;
 extern std::condition_variable cv_test;
 
-extern DataStatus trainReadyFlag;
-extern DataStatus valReadyFlag;
-extern DataStatus testReadyFlag;
+#define SET_VALIDATION(val)                                              \
+  do {                                                                   \
```
Comment (jijoongmoon) [inline nntrainer/src/databuffer_func.cpp:56]: ditto

**Review by jijoongmoon — CHANGES_REQUESTED** [review]
need to check.

File: nntrainer/include/databuffer_util.h (hunk 1 above)
Comment (kparichay) [reply nntrainer/include/databuffer_util.h]: updated.

File: nntrainer/src/databuffer_file.cpp (hunk 2 above)
Comment (kparichay) [reply nntrainer/src/databuffer_file.cpp]: yes, removed.

File: nntrainer/src/databuffer_func.cpp (hunk 3 above)
Comment (kparichay) [reply nntrainer/src/databuffer_func.cpp]: removed.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**